glowing-waffles/photometry.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import print_function, division, absolute_import
+import numpy as np
+from astropy import units as u
+from astropy.stats import mad_std
+from astropy.io import fits
+from astropy.table import Table, join
+from photutils import aperture_photometry, CircularAperture, CircularAnnulus
+from photutils.background import Background
+
+
+def photutils_stellar_photometry(ccd_image, sources, aperture_radius,
+                                 annulus_radius, #coord_type = pixel,
+                                 bkgd_est_fcn = mad_std):
+    """
+    Perform aperture photometry on an image, with a few options for estimating
+    the local background from an annulus around the aperture.
+
+    Parameters
+    ----------
+    ccd_image : CCDData object
+        Image on which to perform aperture photometry.
+
+    sources : astropy Table object
+        Table of extracted sources. Assumed to be the output of
+        photutils.daofind() source extraction function.
+
+    aperture_radius : float
+        Radius of aperture(s) in pixels. This is also used as
+        inner radius for annulus object.
```

Comment:
The inner radius for the annulus needs to be larger than the radius of the aperture so that the annulus does not include much of the light from the star itself.

You could either add another argument for the inner aperture radius or make annulus radius a tuple of inner radius, outer radius